### PR TITLE
new planck keymap, new feature - hybrid shift/enter action key (great for small keyboards!)

### PR DESCRIPTION
**Pull request by BigTuna94**

### CODE REVIEW

File: docs/feature_space_shift_cadet.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Essentially, you hit the Shift on its own, and it acts as the enter key. When hit with other keys, the Shift key keeps working as it always does. Yes, it's as cool as it sounds. This solution works better than using a macro since the timers defined in quantum allow us to tell when another key is pressed, rather than just having a janky timer than results in accidental endlines. 
+
+To use it, use `KC_SFTENT` (Shift, Enter) for any Shift on your keymap.
+
+It's defaulted to work on US keyboards, but if you'd like to use a different key for Enter, you can define those in your `config.h` like this:
+
+    #define SFTENT_KEY KC_ENT
+
+
+The only other thing you're going to want to do is create a `Makefile` in your keymap directory and set the following:
```
Comment (jackhumbert) [inline docs/feature_space_shift_cadet.md]: `Makefiles` are actually ignored now! This will need to be the `rules.mk` file :)

Comment (BigTuna94) [reply docs/feature_space_shift_cadet.md]: Ah, I changed my personal Makefiles to a rules.mk file, but I missed that in the documentation. Uploading the change to doc now. Thanks @jackhumbert!

**Comment by BalzGuenat** [comment]
May I ask how this is different from simply using `RSFT_T(KC_ENT)`?

**Comment by BigTuna94** [comment]
Really the only difference is that the shift key up is executed before
enter key down. This actually makes a significant impact on how enter is
interpreted in most applications.

**Comment by jackhumbert** [comment]
Awesome! That is an interesting distinction :) thanks!

**Pull request merged**